mlir/include/mlir/Transforms/Passes.td

```diff
@@ -268,8 +268,10 @@ def Inliner : Pass<"inline"> {
   let summary = "Inline function calls";
   let constructor = "mlir::createInlinerPass()";
   let options = [
-    Option<"defaultPipelineStr", "default-pipeline", "std::string",
-           /*default=*/"\"canonicalize\"", "The default optimizer pipeline used for callables">,
+    Option<"preInlineCallableOptPipelineStr", "pre-inline-pipeline",
+           "std::string", /*default=*/"\"canonicalize\"",
+           "The optimizer pipeline used for callables that do not have "
+           "a dedicated optimizer pipeline in opPipelineList">,
     ListOption<"opPipelineList", "op-pipelines", "OpPassManager",
                "Callable operation specific optimizer pipelines (in the form "
                "of `dialect.op(pipeline)`)">,
```

Comment:
I see why the original name was "defaultPipeline", it's because the user can also specify another pipeline per-operation.

I would likely keep the original name then and not change the option here.

These options should be documented carefully in the InlinerConfig as well.

Reply:
Okay, I will rename it back.